src/renderer/components/Lofi/Cover/TrackInfo/style.scss

```diff
@@ -33,6 +33,7 @@
 .track-info {
     opacity: 0;
     font-size: 90%;
+    pointer-events: none;
```

Comment:
This is a great trick, I usually fix this with event debouncing. Learn something new every day!

Reply:
This also prevents hovering over track-info area and bringing it up unintentionally while trying to click on something behind that since lofi is always on top. Seemed like two birds one stone 😄